README.rst

```diff
@@ -4,53 +4,114 @@ TruFont
 =======
 
 `TruFont <https://trufont.github.io>`__ is a font-editing application
-written with Python3, ufoLib, defcon and PyQt5.
+written with Python3, ufoLib, defcon, and PyQt5.
+
+.. image:: Lib/trufont/resources/screenshot.png
 
 Getting started
-~~~~~~~~~~~~~~~
+===============
+
+Quick installation instructions
+-------------------------------
+
+If you have Python3 and Git installed and want to quickly install 
+and run TruFont in a virtual environment, open a terminal and enter 
+the following chain of commands as one line:
+
+``python3 -m venv trufont-venv && source trufont-venv/bin/activate && cd trufont-venv && git clone https://github.com/trufont/trufont.git && cd trufont && pip install --upgrade -r requirements.txt && pip install --editable . && trufont``
+
+Or, as separate lines, making modifications when needed:
+
+   .. code::
+
+     python3 -m venv trufont-venv
+     source trufont-venv/bin/activate
+     cd trufont-venv
+     git clone https://github.com/trufont/trufont.git
+     cd trufont
+     pip install --upgrade -r requirements.txt
+     pip install --editable .
+     trufont
+
+Detailed Installation Instructions
+----------------------------------
 
 1. Install **Python 3.5** (or later):
 
    -  OS X: Install using `Homebrew <http://brew.sh/>`__:
       ``brew install python3``
    -  Windows: Download installer from 
       `python.org/downloads <https://www.python.org/downloads/>`__
-   -  Linux: It's usually packaged with the OS.
+   -  Linux: It's usually packaged with the OS,
+      run ``python3 --version`` to see what you have installed. 
 
-2. Set up a new Python **virtual environment**. Although this is not
-   required, it's *highly recommended*, since TruFont has lots of
+2. Install **Git 2.1** (or later):
```

Comment:
Perhaps, `Install **Git** (v2.1+):`?

Also perhaps add a line to say how to check if you have it already?